Firestore/Swift/Tests/Codable/FirestoreEncoderTests.swift

```diff
@@ -37,7 +37,7 @@ private func assertEncodes<X: Equatable & Codable>(_ model: X, to expected: [Str
 
 private func assertDecodes<X: Equatable & Codable>(_ model: [String: Any], to expected: X) -> Void {
   do {
-    let decoded = try Firestore.Decoder().decode(X.self, from: model)
+    let decoded = try Firestore.Decoder().decode(X.self, from: model, forDoc: FSTTestDocRef("abc/123"))
```

Comment:
Make sure to drop the documents that are irrelevant once you make the document reference optional.